### PR TITLE
AB#3926 -- Updated adult confirm page from figma

### PR DESCRIPTION
**Pull request by AdamP-HRSDC-RHDCC**
### Description
Updated both en/fr for msca section

### Related Azure Boards Work Items
[AB#3926](https://dev.azure.com/DTS-STN/1fc40a8f-28cf-47bc-b6e4-1c234bd06177/_workitems/edit/3926)

### Screenshots (if applicable)
image

### CODE REVIEW

File: frontend/public/locales/en/apply-adult.json [hunk 1]
```diff
@@ -60,9 +60,13 @@
     "use-code-one-week": "You can use your application code to track the progress of your application one week after applying.",
     "mail-letter": "Once your application is processed, we will mail you a letter to inform you whether your application has been accepted or rejected.",
     "register-msca-title": "Register for a My Service Canada Account",
-    "register-msca-text": "You will be able to view a copy of the Canadian Dental Care Plan letters mailed to you in <mscaLinkAccount>My Service Canada Account (MSCA)</mscaLinkAccount>. <mscaLinkCreate>Create an account</mscaLinkCreate> so you can view your letters once the application has been processed.",
+    "register-msca-text": "Prepare to get important updates about your Canadian Dental Care Plan application by registering for a <mscaLinkAccount>My Service Canada Account (MSCA).</mscaLinkAccount>",
```
Comment (sebastien-comeau) [inline frontend/public/locales/en/apply-adult.json:63]: Period `.` must be outside the link.

**Review by sebastien-comeau — CHANGES_REQUESTED** [review]

File: frontend/public/locales/en/apply-adult.json (hunk 1 above)
Comment (AdamP-HRSDC-RHDCC) [reply frontend/public/locales/en/apply-adult.json]: Moved period

File: frontend/public/locales/en/apply-adult.json [hunk 2]
```diff
@@ -60,9 +60,13 @@
     "use-code-one-week": "You can use your application code to track the progress of your application one week after applying.",
     "mail-letter": "Once your application is processed, we will mail you a letter to inform you whether your application has been accepted or rejected.",
     "register-msca-title": "Register for a My Service Canada Account",
```
[the remark this answers is not in the document]
Comment (AdamP-HRSDC-RHDCC) [reply frontend/public/locales/en/apply-adult.json]: updated title

File: frontend/public/locales/fr/apply-adult.json [hunk 3]
```diff
@@ -60,9 +60,13 @@
     "use-code-one-week": "Vous pouvez utiliser votre code de demande pour suivre la progression de votre candidature une semaine après avoir postulé.",
     "mail-letter": "(FR) Once your application is processed, we will mail you a letter to inform you whether your application has been accepted or rejected.",
     "register-msca-title": "Inscrivez-vous à Mon dossier Service Canada dès maintenant",
-    "register-msca-text": "(FR) You will be able to view a copy of the Canadian Dental Care Plan letters mailed to you in <mscaLinkAccount>My Service Canada Account (MSCA)</mscaLinkAccount>. <mscaLinkCreate>Create an account</mscaLinkCreate> so you can view your letters once the application has been processed.",
+    "register-msca-text": "Préparez-vous à recevoir des mises à jour importantes concernant votre demande d'adhésion au Régime canadien de soins dentaires en vous inscrivant à <mscaLinkAccount>Mon dossier Service Canada (MDSC).</mscaLinkAccount>",
```
[the remark this answers is not in the document]
Comment (AdamP-HRSDC-RHDCC) [reply frontend/public/locales/fr/apply-adult.json]: Moved period

File: frontend/public/locales/fr/apply-adult.json [hunk 4]
```diff
@@ -60,9 +60,13 @@
     "use-code-one-week": "Vous pouvez utiliser votre code de demande pour suivre la progression de votre candidature une semaine après avoir postulé.",
     "mail-letter": "(FR) Once your application is processed, we will mail you a letter to inform you whether your application has been accepted or rejected.",
     "register-msca-title": "Inscrivez-vous à Mon dossier Service Canada dès maintenant",
```
[the remark this answers is not in the document]
Comment (AdamP-HRSDC-RHDCC) [reply frontend/public/locales/fr/apply-adult.json]: updated title

File: frontend/app/routes/$lang/_public/apply/$id/adult/confirmation.tsx [hunk 5]
```diff
@@ -227,7 +227,15 @@ export default function ApplyFlowConfirm() {
       <section>
         <h2 className="font-lato text-3xl font-bold">{t('confirm.register-msca-title')}</h2>
         <p className="mt-4">
-          <Trans ns={handle.i18nNamespaces} i18nKey="confirm.register-msca-text" components={{ mscaLinkAccount, mscaLinkCreate }} />
+          <Trans ns={handle.i18nNamespaces} i18nKey="confirm.register-msca-text" components={{ mscaLinkAccount }} />
+        </p>
+        <p className="mt-4">{t('confirm.register-msca-info')}</p>
+        <ul className="ml-6 list-disc">
```
[the remark this answers is not in the document]
Comment (AdamP-HRSDC-RHDCC) [reply frontend/app/routes/$lang/_public/apply/$id/adult/confirmation.tsx]: updated className

**Pull request merged**